invenio_accounts/validators.py

```diff
@@ -95,3 +101,30 @@ def validate_email(email):
                 _("Supplied email address %(x_addr)s is invalid.",
                   x_addr=email)
             )
+
+
+def validate_password(password):
+    """Validate password."""
+    min_length = cfg['CFG_ACCOUNT_MIN_PASSWORD_LENGTH']
+    if len(password) < min_length:
+        raise validators.ValidationError(
+            _("Password must be at least %(x_pass)d characters long.",
+              x_pass=min_length))
+
+
+def validate_nickname_is_available(nickname):
+    """Check whether the nickname is taken."""
+    try:
+        User.get_by_nickname(nickname)
+    except NoResultFound:
+        return
+    raise validators.ValidationError('nickname %s is not available' % nickname)
```

Comment:
Please make the user facing validation error translatable (use `%(x_name)s` instead of plain `%`).